server/service/src/invoice/inventory_adjustment/add_new_stock_line/validate.rs

```diff
@@ -0,0 +1,45 @@
+use repository::StorageConnection;
+
+use crate::common_stock::check_stock_line_does_not_exist;
+
+use crate::stocktake_line::validate::{check_active_adjustment_reasons, check_reason_is_valid};
+use crate::validate::check_store_exists;
+
+use super::insert::{AddNewStockLine, AddNewStockLineError};
+
+pub fn validate(
+    connection: &StorageConnection,
+    store_id: &str,
+    input: &AddNewStockLine,
+) -> Result<(), AddNewStockLineError> {
+    use AddNewStockLineError::*;
+    if !check_store_exists(connection, store_id)? {
```

Comment:
Hmm, you probably got this one from stocktake ? 

There is auth check to make sure store exists, and there would be a conflict in DB if it doesn't exist, I dont think this test is really needed